ceph/ceph_admin/add.py

```diff
@@ -48,4 +48,7 @@ def add(self: DaemonProtocol, config: Dict):
 
             base_cmd.append(host)
 
+        if config.get("args"):
```

Comment:
Can you please add this new support for args in add.py to its doc string's example.